.github/workflows/fetch-and-ingest-gisaid-master.yml

```diff
@@ -24,18 +24,24 @@ jobs:
     - name: setup
       run: ./bin/setup-github-workflow
 
-    - name: ingest
+    - name: run_pipeline
       run: |
         ./bin/write-envdir env.d \
           AWS_DEFAULT_REGION \
           GISAID_API_ENDPOINT \
           GISAID_USERNAME_AND_PASSWORD \
-          GITHUB_REF \
           GITHUB_RUN_ID \
           SLACK_TOKEN \
           SLACK_CHANNELS \
           PAT_GITHUB_DISPATCH
 
+        GITHUB_BRANCH=${GITHUB_REF#refs/heads/}
+        declare -a config
+        config+=(
+          fetch_from_database=True
+          trigger_preprocessing=True
```

Comment:
Pre-merge change: Is there a reason these config arguments aren't hardcoded in the Snakemake command below? I'd think it would be more readable to keep these config flags as close as possible to where they are used.

Reply:
We [recently shifted to this form in nextstrain/ncov](https://github.com/nextstrain/ncov/commit/c91ad66fa22cedc7484c72c8b67336a35ef562f3) which I actually find easier to read, especially if we increase the complexity of the actions in the future and start modifying which parameters are in `config`. (I don't feel super strongly about this however.)